source/core/src/main/com/csse3200/game/screens/MainMenuScreen.java

```diff
@@ -79,26 +79,25 @@ private void loadAssets() {
     logger.debug("Loading assets");
     ResourceService resourceService = ServiceLocator.getResourceService();
     resourceService.loadTextures(mainMenuTextures);
-    ServiceLocator.getResourceService().loadAll();
+    resourceService.loadAll();
   }
 
   private void unloadAssets() {
     logger.debug("Unloading assets");
     ResourceService resourceService = ServiceLocator.getResourceService();
     resourceService.unloadAssets(mainMenuTextures);
   }
-
   /**
    * Creates the main menu's ui including components for rendering ui elements to the screen and
    * capturing and handling ui input.
    */
   private void createUI() {
-    logger.debug("Creating ui");
+    logger.debug("Creating UI");
     Stage stage = ServiceLocator.getRenderService().getStage();
     Entity ui = new Entity();
     ui.addComponent(new MainMenuDisplay())
-        .addComponent(new InputDecorator(stage, 10))
-        .addComponent(new MainMenuActions(game));
+            .addComponent(new InputDecorator(stage, 10))
```

Comment:
again spacing looks like auto-formatter issue, we should create a branch separately to run an auto-formatter that the class agrees upon in the next studio.